templates/index.html

```diff
@@ -200,7 +199,7 @@ <h6>Current Position</h6>
                     </div>
 
                     <div class="custom-btn-group mt-4">
-                        <a href="/stories" class="btn custom-btn custom-btn-bg custom-btn-link">Show More Stories</a>
+                        <a href="/stories" class="btn custom-btn custom-btn-bg custom-btn-link">See More Stories</a>
```

Comment:
"Show More Stories" gave me the feeling that more stories would render on that page rather than redirect the user. Updated wording.